lib/css/exports/_stacks-constants-helpers.less

```diff
@@ -21,18 +21,20 @@
 //      Used for a bird's eye view of components dependent on the z-axis.
 //      Try to avoid customizing these.
 //  ----------------------------------------------------------------------------
-@zi-hide:               -1;     //  Hide something all the elements
-@zi-base:               0;      //  Reset to 0
-@zi-selected:           25;     //  Pop above siblings
-@zi-active:             30;     //  Pop above selected siblings
-@zi-dropdown:           1000;   //  Dropdowns
-@zi-popovers:           2000;   //  Popovers, Popups, Autocompletes
-@zi-tooltips:           3000;   //  Tooltips
-@zi-banners:            4000;   //  Banners
-@zi-navigation:         5000;   //  Navigation Bars
-@zi-navigation-fixed:   5050;   //  Fixed navigation bars
-@zi-modals:             9000;   //  Modals
-@zi-modals-background:  @zi-modals - 50;
+body {
```

Comment:
Some discussion might need to be had about where we actually put some of these. I don't imagine a theme or a user modifying these specific variables, so they could be tied to `:root`? 🤔 

This works for now, of course.